wiki/Rules/el.md

```diff
@@ -0,0 +1,79 @@
+
+1.  Μην δημιουργείτε threads στα [Pending Submissions](/wiki/Beatmaps) και [WIP](/wiki/Glossary) subforum. Χρησιμοποιήστε το κατάλληλο subforum όταν δημιουργείτε ένα thread. (Αυτό προφανώς αποκλείει τα BSS threads.)
+2.  Μην βγαίνεται εκτός θέματος στα BSS threads. Κάθε BSS thread αφορά ένα συγκεκριμένο beatmap. Αν έχετε κάποιο πρόβλημα που δεν σχετίζεται άμεσα με αυτό, κάντε το post στο κατάλληλο forum.
+3.  Μην κλέβετε beatmap αλλωνών, παρουσιάζοντάς τα σαν να είναι δικά σας. Επίσης, μην ανεβάζετε και μην χρησιμοποιήσετε έργα αλλωνών χωρίς τη ρητή άδειά τους (συμπεριλαμβανομένoυ, αλλά όχι περιορισμένα σε skins και guest difficulties).
+
+Off-Topic Κανόνες
+-----------------
+
+1.  Στο Off-Topic είναι εκεί που "πεθαίνουν" τα "κακά" threads. Τίποτα αξιοσημείωτο δεν βρίσκεται σε αυτό το subforum. Αν το θέμα σας έχει κάποια αξία, τότε δεν ανήκει στο off-topic. Αν το θέμα σας μεταφέρθηκε στο off-topic από κάποιο άλλο μέρος, σημαίνει πως έχει κριθεί ανάξιο σοβαρής συζήτησης.
+2.  Η υπερβολική χαζομάρα απαγορεύεται σε μεγάλο βαθμό. Εδώ εφαρμόζεται ο κανόνας \#3 της κοινότητας. Είναι εντάξει να αστειεύεσται από 'δώ και από 'κεί - αλλά όχι να γράφετε συνέχεια ανοησίες.
+3.  Τα threads γενεθλίων επιτρέπονται ΜΟΝΟ στο Off-topic, και πουθενά αλλού.
+
+In-game Chat Κανόνες
+--------------------
+
+1.  Απαγορεύεται το spam.
+2.  Σε περίπτωση που βρεθείτε να προσπαθείτε να αποφύγετε silences, θα οδηγηθείτε άμεσα σε περιορισμό, και θα βρεθείτε.
+    1.  Το να μιλάτε για silences (δικό σας προηγούμενο ή κάποιου άλλου) σε δημόσιες συζητήσεις απαγορεύεται και τιμωρείται με silence.
+3.  Δημόσια συζήτηση για NSFW περιεχόμενο ή οτιδήποτε σεξουαλικό δεν επιτρέπεται σύμφωνα με το κανόνα #5 της κοινότητας. Αυτός ο κανόνας δεν ισχύει για τα προσωπικά μηνύματα.
+4.  Όλοι οι σύνδεσμοι (links) με εξωτερικό περιεχόμενο (εικόνες, threads, βίντεο κ.λπ.) δεν πρέπει να περιέχουν φωτοευαίσθητο περιεχόμενο (γρήγορο φλας κ.λπ.).
+5.  Η πλαστοπροσωπία απαγορεύεται ρητά, ειδικά του προσωπικού (Staff).
+6.  Μη διαφημίζετε multiplayer δωμάτια πέρα από το #lobby.
+7.  Απαγορεύονται ρητά οι συζητήσεις περί cheating, hacking, ή οποιαδήποτε άλλη μορφή κακομεταχείρισης του client.
+    1.  Απαγορεύεται επίσης η αναφορά και η συζήτηση απαγορευμένων (banned) παικτών.
+8.  Σε κανάλια συγκεκριμένων γλωσσών, μιλήστε στην κυρίαρχη γλώσσα εάν συμμετέχετε σε μία συζήτηση. Όταν σε ένα κανάλι δεν αναφέρεται μια συγκεκριμένη γλώσσα, τότε είναι πάντα τα Αγγλικά.
+    1.  Οι Συντονιστές (moderators) μπορούν να μιλήσουν Αγγλικά σε κανάλια συγκεκριμένων γλωσσών, σε περίπτωση που κριθεί απαραίτητο, για να χειριστούν μια κατάσταση κατάλληλα.
+    2.  Σε όλα τα άλλα κανάλια (#osu, #english, #modhelp, #modreqs, #videogames, #ctb, #taiko, #osumania), απαγορεύεται να μιλάτε οποιαδήποτε άλλη γλώσσα εκτός από τα Αγγλικά.
+    3.  Τα κανάλια #lobby και #help είναι πολύγλωσσα, επομένως δεν υπάρχουν περιορισμοί στην γλώσσα.
```

Comment:
Hmm, this part seems to have been removed in the English Rules, but I guess I'll keep it. However, it seems to be written this way \#channel_name . Which one would be better?